validator-api/src/main.rs

```diff
@@ -232,6 +241,15 @@ fn override_config(mut config: Config, matches: &ArgMatches) -> Config {
         config = config.with_epoch_length(Duration::from_secs(epoch_length * 60 * 60));
     }
 
+    if let Some(monitor_threshold) = matches
+        .value_of(REWARDING_MONITOR_THRESHOLD_ARG)
+        .map(|t| t.parse::<u8>())
+    {
+        let monitor_threshold =
+            monitor_threshold.expect("Provided monitor threshold is not a number!");
```

Comment:
We could also check the `0-100` value range here

Reply:
Why 0-100 range?

Reply:
That's what I saw being mentioned in the template documentation